prescriptions/bl_/black/click8x10x0_black22x1x0.yaml

```diff
@@ -0,0 +1,23 @@
+units:
+  steps:
+  - name: Click8x10x0Black22x1x0Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+          name: click
+          version: <8.10.0
```

Comment:
```suggestion
          version: <8.1.0
```

Reply:
Thanks! :100: Fixed in https://github.com/thoth-station/prescriptions/pull/23484/commits/8335af62bcef18968046f4064601f23c360f58f3